### ISSUE TITLE
BUG - Telegram integration image centralize and size

### ISSUE THREAD

**Issue by itsalb3rt** [ta-vivo/ta-vivo#120]
#### Issue description
We need to add a size for the image of telegram integration example and centralize the image

![image](https://github.com/ta-vivo/ta-vivo/assets/35310226/26c09d23-ed63-4d0d-a255-fdb20de77565)

#### Steps to reproduce the issue

1.  Go to `/integrations/add` route
2. Push to the Telegram integration
3. See the example of the integration on the bottom of the page

#### What's the expected result?

Be creative  ☺️

#### What's the actual result?

-

**Comment by rafieltq** [comment]
I'd love to take this issue, but i don't get the situation well, can you further explain please?

**Comment by itsalb3rt** [comment]
Yes sure. The image alignment is set by default to the left side, but I think this looks better is centralized.

About the size, the image doesn't have a specific size, I think this work better is we add a size like 

```
<Image style="max-width: 500px" />
```

**Comment by robertlluberes** [comment]
The issue can be considered closed since it has been fixed by PR #122.

**Comment by itsalb3rt** [comment]
Thanks @robertlluberes 🤝
